app/controllers/likes_controller.rb

```diff
@@ -1,31 +1,21 @@
 class LikesController < ApplicationController
   class LikesController < ApplicationController
-    before_action :authenticate_user! # Add authentication if needed
+    before_action :authenticate_user!
 
-    # Create a new like for a resource (e.g., a post)
     def create
-      @like = Like.new(like_params)
-      @like.user = current_user # Assign the current user as the one who liked
+      @like = @post.likes.build(user: @current_user)
 
       if @like.save
-        redirect_to @like.liked_item, notice: 'Like was successfully created.'
+        redirect_to @post, notice: 'Liked the post!'
       else
-        render :new
+        redirect_to @post, alert: 'Unable to like the post.'
       end
     end
 
-    # Remove a like for a resource
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you make your own logo of your name and put that in the readme file at the top to present yourself as a brand in the job market or tech industry to catch the recruiters' eyes 👍